Kickstarter-iOS/AppDelegate.swift

```diff
@@ -207,14 +208,14 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
 
     self.window?.tintColor = .ksr_green_700
 
-    self.viewModel.inputs.applicationDidFinishLaunching(
```

Comment:
I *think* we still need to call `self.viewModel.inputs.applicationDidFinishLaunching` somewhere in here - it's used for a bunch of stuff in `AppDelegateViewModel` 🤔

Reply:
lol yes good catch - that would've broke things